README.md

```diff
@@ -73,27 +73,27 @@ by:
 
 ```bash
 ...do some change and commit...
-$ git push-stack
+$ mrgfy push-stack
```

Comment:
```suggestion
$ mergify push-stack
```
I don't think we want to truncate the name here. It's easy to `alias` for anyone if they want too.

Reply:
I will take this comment in account, the others are not relevant since I totally re-write readme.me in the last commit of the stack. I should have removed it from this commit and that is what I will do.